nats-streaming-http-connector/README.md

```diff
@@ -13,12 +13,11 @@ The job of the connector is to read messages from the subject, call an HTTP endp
 - `CONTENT_TYPE`: Content type used while creating post request
 - `NATS_SERVER`: NATs connection string, like `nats://127.0.0.1:4222`
```

Comment:
```suggestion
- `NATS_SERVER`: NATS server address . It can be a remote address  `nats://127.0.0.1:4222` or in case deployed in Kubernetes, reached by it's FQDN: `nats://nats:4222`
```

Reply:
Thanks changed as per suggestion